client/my-sites/woocommerce/index.js

```diff
@@ -29,9 +30,9 @@ function setup( context, next ) {
 	const site = getSelectedSiteWithFallback( state );
 	const siteId = site ? site.ID : null;
 
-	// Only allow AT sites to access, unless the woop feature flag is enabled.
+	// Only allow AT and Simple Business sites to access, unless the woop feature flag is enabled.
 	// todo: remove redirect and rely on plan eligibility checks in the landing page component
-	if ( ! isEnabled( 'woop' ) && ! isAtomicSite( state, siteId ) ) {
+	if ( isEnabled( 'woop' ) && ! isAtomicSite( state, siteId ) && ! isBusiness( site.plan ) ) {
```

Comment:
To stay in sync [with the sidebar nudge](https://github.com/Automattic/jetpack/pull/22005).